code/modules/vehicles/mecha/mecha_actions.dm

```diff
@@ -161,3 +161,36 @@
 		if(!istype(chassis.equip_by_category[i], /obj/item/mecha_parts/mecha_equipment))
 			continue
 		INVOKE_ASYNC(chassis.equip_by_category[i], TYPE_PROC_REF(/obj/item/mecha_parts/mecha_equipment, attempt_rearm), owner)
+
+/datum/action/vehicle/sealed/mecha/repairpack
+	name = "Use Repairpack"
+	action_icon_state = "mech_damtype_toxin" // todo kuro needs to make an icon for this
```

Comment:
hi todo

Reply:
Yea still a todo he said he'll do sprites for V2 of mech readd